{
  "module": "Test",
  "kind": "Generic Error",
  "text": [
    { "type": "text", "content": "Wrong number of arguments. Expected " },
    { "type": "code", "content": "2" },
    { "type": "text", "content": " and got " },
    { "type": "code", "content": "1" },
    { "type": "text", "content": "." }
  ],
  "span": {
    "filename": "test-programs/suites/010-functions/006-wrong-arity/Test.aum",
    "startp": { "line": 7, "column": 8 },
    "end": { "line": 7, "column": 17 }
  },
  "context": [
    [ 5, "" ],
    [ 6, "    function main(): ExitCode is" ],
    [ 7, "        foo(nil);" ],
    [ 8, "        return ExitSuccess();" ],
    [ 9, "    end;" ]
  ]
}